nucypher/characters/control/specifications/fields/treasuremap.py

```diff
@@ -39,13 +35,17 @@ def _deserialize(self, value, attr, data, **kwargs):
             raise InvalidInputData(f"Could not parse {self.name}: {e}")
 
     def _validate(self, value):
-
-        splitter = BytestringSplitter(signature_splitter,
-                                  (bytes, HRAC_LENGTH),  # hrac
-                                  (UmbralMessageKit, VariableLengthBytestring)
-                                  )  # TODO: USe the one from TMap
         try:
-            signature, hrac, tmap_message_kit = splitter(value)
-            return True
-        except InvalidNativeDataTypes as e:
-            raise InvalidInputData(f"Could not parse {self.name}: {e}")
+            # Unsigned TreasureMap (Federated)
+            from nucypher.policy.maps import TreasureMap as UnsignedTreasureMap
+            splitter = UnsignedTreasureMap.get_splitter(value)
+            splitter(value)
+        except InvalidNativeDataTypes:
+            try:
```

Comment:
This nested try-except will be improved in the Porter EPIC, but knowing the context (federated or not) when this validation is performed.